NoSqlAssignment/blog_schema.doc

```diff
@@ -0,0 +1,88 @@
+
+Following schema is for collection posts:
+
+{
+	_id:post_id ,
+	post_heading : "Title of the post" ,
```

Comment:
Shouldn't add the "post"  as a prefix to your keys if the key belongs to the same collection. This is repetition.